playbooks/robusta_playbooks/krr.py

```diff
@@ -311,7 +412,9 @@ def krr_scan(event: ExecutionBaseEvent, params: KRRParams):
         subject = event.get_subject()
         args_sanitized = format_event_templated_string(subject, params.args_sanitized)
 
-    python_command = f"python krr.py {params.strategy} {args_sanitized} {additional_flags} "
+    publish_scan_args = f"--publish_scan_url={KRR_PUBLISH_URL} --scan_id={scan_id} --start_time=\"{datetime.now()}\""
```

Comment:
Last thing. Do we really want to pass the start_time and not just actually take it from the krr.job start time?

For example the job could be pending for couple of minutes before actually start WDYT?